WooCommerce/Classes/ViewRelated/Authentication/AccountCreationForm.swift

```diff
@@ -123,13 +123,13 @@ struct AccountCreationForm: View {
                 Button(Localization.submitButtonTitle) {
                     Task { @MainActor in
                         isPerformingTask = true
-                        let result = await viewModel.createAccount()
-                        isPerformingTask = false
-                        switch result {
-                        case .success:
+
+                        do {
+                            try await viewModel.createAccount()
```

Comment:
Perhaps we can use `try?` here to simplify this block if we want to ignore the error case